### DISCUSSION TITLE
Anfahren im Standgas

### DISCUSSION THREAD

**Discussion by ChrisJD8400R** [Mogli12/KeyboardSteer#803] [Ideas]
Ich habe bemerkt, dass beim anfahren im Standgas (kleiner Gang) der schlepper nicht eigenstädig losfährt. Man muss immer die Drehzahler des Motors erhöhen damit der Schlepper losfährt.

MfG Chris

**Comment by Mogli12** [comment]
Ja, das ist korrekt so. Unterhalb von 3,6 km/h greift die automatische Bremse vom Spiel. Daran ändert VCA nichts.

**Reply by A3Ambition** [reply]
Hallo Zusammen, vielleicht könnte VCA daran ja etwas ändern? ;-)

Joke aside…

Ich weiß ich hatte das Thema schon einmal angesprochen aber da wir nun schon 2 Leute mit dem selben „Realismus“-Empfinden sind besteht evtl. mehr Interesse an der Thematik.

Viele Grüße.

**Comment by Mogli12** [comment]
Probiere mal Version 105. Damit sollte es etwas besser sein.

**Reply by A3Ambition** [reply]
Wird heute Abend getestet. Vielen Dank schon einmal für die Rückmeldung!

**Reply by A3Ambition** [reply]
Hallo Stefan,

die Implementierung ist einfach nur Weltklasse! Fahrzeuge mit manuellen Getrieben fahren sich nun sehr realistisch - vollkommen neues Spielgefühl! DANKE!

Getestet habe ich so gut wie alle Gertriebevariationen. Ein kleines Problem scheint es jedoch noch bei den stufenlosen Getrieben zu geben: hier ist die Autobrake unter 3.6 Km/h noch immer aktiv. Hier tritt zusätzlich das Phänomen auf dass die Autobrake sich aber wieder löst bei 0.x Km/h und das Fahrzeug dann eben mit dieser sehr geringen Geschwindigkeit weiter rollt. Bekommt man hier die Autobrake vielleicht auch komplett raus?

Hoffe das Feedback passt…

**Comment by Mogli12** [comment]
Hallo,

auch die stufenlosen Getriebe von Giants haben einen Bereich, den sie abdecken. Unterhalb von 1 km/h wird dann auch eine Art Kupplung geöffnet. Daher kann man das Fahrzeug nicht einfach ausrollen lassen.

Viele Grüße
